core/src/main/java/org/openstack4j/api/Builders.java

```diff
@@ -744,6 +746,15 @@ public static DomainBuilder domain() {
         return KeystoneDomain.builder();
     }
 
+    /**
+     * The builder to create an Encryption Type
+     *
+     * @return the encryption type builder
+     */
+    public static EncryptionTypeBuilder encryptionType() {
+        return CinderEncryptionType.builder();
```

Comment:
This class is quite nasty as it is and needs to be refactored. However, expression `Builders.encryptionType()` lacks so much contextual information.

For the call to be intuitive, we either have to be more explicit (`Builders.blockVolumeTypeEncryption()`) or, and that is what I prefer, introduce one similar "service sub-builder" like `NetworkBuilder` so one would call `Builders.blockStorage().volumeTypeEncryption()`. What do you think?

Reply:
Yes I agree, modified code as suggested.